gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-bin.zip
+networkTimeout=10000
```

Comment:
Why has this been added? Irrespective of what this does, documentation seems to imply that 10000 is its default value anyways: https://docs.gradle.org/current/userguide/gradle_wrapper.html

Reply:
I think it was added due to the gradle update and seemingly wasn't automatically removed through downgrading  it back again to 7.6. We will removed it.